Tasks/google-play-rollout-update/task.json

```diff
@@ -14,7 +14,7 @@
     ],
     "version": {
         "Major": "1",
```

Comment:
The other tasks had a major bump, is this okay to just have a minor bump?

Reply:
Yes, this one was already using V3 of the google APIs (a issue from last year), so the UI didn't need to change.  So, a minor version is all that's necessary.

Reply:
So, I released that there is a gap in functionality that required me to update this afterall.  We need a way to increase the rollout fraction in _any_ track, so I added 'track' and bumped the version here.